pnpm-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+packages:
+  - 'charts/**'
+  - '!**/octopus-deploy'
```

Comment:
would this be removed when/if we change the octopus deploy helm chart to use versioning?

Reply:
Yep, for now it's just to stop an unexpected behaviour but would be removed when there is a package.json in that folder.